clex/driver.c

```diff
@@ -400,7 +400,13 @@ void reverse_toks (int idx)
   int matched = 0;
   int which = 0;
   int i;
+
+#ifdef _MSC_VER
+  int *saved = calloc(N, sizeof(int));
```

Comment:
this program is so short-lived that calls to free() are almost certainly not necessary either (let's leave them in though)